rules-tests/DeadCode/Rector/Assign/RemoveUnusedVariableAssignRector/Fixture/skip_used_in_expr.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\Assign\RemoveUnusedVariableAssignRector\Fixture;
+
+class SkipUsed
+{
+    public function run()
+    {
+        // run file node only if
+        $fileNode = new FileNode($smartFileInfo, $parsedStmtsAndTokens->getNewStmts());
+        $result = $this->rectorNodeTraverser->traverseFileNode($fileNode);
```

Comment:
Shouldn‘t the `$result =` be deleted in this case?

Reply:
This rule removal is remove the `Assign` part, not the variable, so it is kept.

Reply:
cleaned up